web-ui/src/main/resources/catalog/templates/editor/accessManager.html

```diff
@@ -0,0 +1,99 @@
+<div class="container" id="gn-access-manager-container">
+  <br/>
+  <br/>
+  <br/>
+  <div class="row">
+    <div class="col-sm-12">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span data-translate="">accessManagerTitle</span>
+        </div>
+
+        <div class="panel-body" data-ng-controller="GnAccessManagerController">
+
+          <div class="row">
+            <div class="col-md-6">
+              <label class="control-label"
+                     data-translate>searchByTitle</label>
+              <input type="text"
+                     class="form-control"
+                     data-ng-model="params.any"
+                     placeholder="{{'anyPlaceHolder' | translate}}"
+                     aria-label="{{'anyPlaceHolder' | translate}}">
+
+              <br/>
+              <label class="control-label"
+                     data-translate>_groupPublished</label>
+              <select class="form-control" data-ng-model="params.group">
+                <option value=""></option>
+                <option data-ng-repeat="g in groups | orderBy:'name'"
+                        value="{{g.name}}">
+                  {{g.label[lang]|empty:g.name}}
+                </option>
+              </select>
+            </div>
+            <div class="col-md-6">
+              <h4 data-translate>accessManagerChooseGroups</h4>
+              <ul>
+                <li data-ng-repeat="g in groups">
+                  <label>
+                    <input type="checkbox"
+                           data-ng-model="g.selected"
+                           data-ng-change="buildQuery()"/>
+                    <span>{{g.label[lang]|empty:g.name}}</span>
+                  </label>
+                </li>
+              </ul>
+
+              <h4 data-translate>accessManagerChooseOperations</h4>
+              <ul>
+                <li data-ng-repeat="o in operations">
+                  <label>
+                    <input type="checkbox"
+                           data-ng-model="o.selected"
+                           data-ng-change="buildQuery()"/>
+                    <span>{{o.label[lang]|empty:o.name}}</span>
+                  </label>
+                </li>
+              </ul>
+            </div>
+          </div>
+
+
+          <table class="table table-striped">
+            <tr>
+              <th></th>
+              <th data-ng-repeat="g in selectedGroups"
```

Comment:
Same as for the operation columns, I think stacking groups horizontally is not an optimal layout. Obviously it is not possible to view more than 3/4 groups at a time, and operation access cannot be easily compared between groups side-by-side, specially when scrolled down and the headers are not visible anymore.

Reply:
In Ifremer, the main use case is: is a record public, available for geocatalogue.fr (published in the group), and available in the group owner. Making a matrix of all is not really the purpose even if possible.